gno.land/pkg/sdk/vm/keeper_test.go

```diff
@@ -23,7 +23,10 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/store/types"
 )
 
-var coinsString = ugnot.ValueString(10_000_000)
+var (
+	coinsString = ugnot.ValueString(20_000_000)
+	coinsToSend = ugnot.ValueString(1_000_000)
+)
```

Comment:
move them to the functions using them or find a more unique and explicit name.

Reply:
Could you please clarify the purpose and benefits of redefining these variables here? They appear to be reused across multiple test cases in keeper_test.go with the same meaning, so it's not immediately clear why they need to be redefined locally.